### DISCUSSION TITLE
Who's in Miami next week and wants to see Ordinals in an exhibition?  -> Come see the impact this repo has had!

### DISCUSSION THREAD

**Discussion by Bird-NZ** [ordinals/ord#4115] [Show and tell]
@casey  or @raphjaph  (and anyone else reading this) - I'm exhibiting next week in Miami next week as part of a group of artists called NFT Aotearoa, all the way over from New Zealand.
I create generative art and inscribe it on Bitcoin -  Come see the impact your work and this repo has had -> register here: https://lu.ma/5sxbb4vw?tk=cSLhDx

I'm showing and releasing 2 collections next week:

**Visualising Bitcoin**
121 unique ordinal inscriptions each built from 100,000,000 sats as points of light. An explosion of digital gold and colour.  A March 2023, sub 600k Ordinal collection.

**Proof of Track**
A collection of photos I've taken while hiking around New Zealand. Morphed with love (& maths) to enhance existing patterns in nature, each inscribed as many metadata packed pieces on the Bitcoin network and reassembled with recursion.

**Comment by raphjaph** [comment]
Please don't advertise events or collections on Github.

**Reply by Bird-NZ** [reply]
Sorry mate, thought as I used this repo and this was the "Show and tell" section folks here may be interested. I'll remove / close the post.
